releng/helm-charts/dirigible/values.yaml

```diff
@@ -14,7 +14,7 @@ image:
   # Repository with Keycloak configuration
   repositoryKeycloak: dirigiblelabs/dirigible-keycloak
   # Pull policy for the repositories
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
For production cases, it’s better to leave it to `IfNotPresent` and alter it manually, if needed for test and development cases.

Reply:
Yes you are right I will keep `IfNotPresent`